module.ts

```diff
@@ -45,7 +48,21 @@ export class Module<T extends Signature> {
 
     constructor(template: Template<T>) {
         this.template = template;
-        this.wasi = new DisabledWASI(this.template.env, this.template.runtime.config.stdout, this.template.runtime.config.stderr);
+
+        const name = `${this.template.identifier}.${uuid()}`
+
+        let stdout = this.template.runtime.config.stdout;
+        let stderr = this.template.runtime.config.stderr;
+
+        if (stdout !== undefined && !this.template.runtime.config.rawOutput) {
```

Comment:
It is discouraged to make direct comparisons with undefined as there are multiple situations in JS where this has unintended consequences (more info here https://eslint.org/docs/latest/rules/no-undefined)
I would rewrite this as `if (typeof stdout !== "undefined"`

Reply:
This isn't the only place this comparison is occurring. I'll change it everywhere then.

Reply:
<img width="744" alt="Screenshot 2023-09-20 at 11 52 03 AM" src="https://github.com/loopholelabs/scale/assets/27162109/a7a558b8-269f-4375-bb16-ae182dc49369">

Looks like doing it with `typeof` does not satisfy the typescript compiler.

Reply:
the result of typeof is a string value, you must compare with `"undefined"`

Reply:
<img width="744" alt="Screenshot 2023-09-20 at 11 53 48 AM" src="https://github.com/loopholelabs/scale/assets/27162109/f23a5e3d-3036-402c-a73d-9d08fb9cbd8f">

Should I be making the change for the `===` side as well?

Reply:
Yes

Reply:
<img width="744" alt="Screenshot 2023-09-20 at 11 57 15 AM" src="https://github.com/loopholelabs/scale/assets/27162109/f94f6ce2-ff3c-4721-bdba-15c657d3a858">

I should modify the signature templates as well right?

Reply:
Indeed